internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -463,44 +504,27 @@ func validateLocalVolumeSizes(api *instance.API, volumes map[string]*instance.Vo
 		}
 	}
 
-	// Get server types.
-	serverTypesRes, err := api.ListServersTypes(&instance.ListServersTypesRequest{
-		Zone: zone,
-	})
-	if err != nil {
-		// Ignore root volume size check.
-		logger.Warningf("cannot get server types: %s", err)
-		logger.Warningf("skip local volume size validation")
-		return nil
-	}
+	volumeConstraint := serverType.VolumesConstraint
 
-	// Validate total size.
-	if serverType, exists := serverTypesRes.Servers[commercialType]; exists {
-		volumeConstraint := serverType.VolumesConstraint
+	// If no root volume provided, count the default root volume size added by the API.
+	if rootVolume := volumes["0"]; rootVolume == nil {
+		localVolumeTotalSize += volumeConstraint.MinSize
+	}
 
-		// If no root volume provided, count the default root volume size added by the API.
-		if rootVolume := volumes["0"]; rootVolume == nil {
-			localVolumeTotalSize += volumeConstraint.MinSize
+	if localVolumeTotalSize < volumeConstraint.MinSize || localVolumeTotalSize > volumeConstraint.MaxSize {
+		min := humanize.Bytes(uint64(volumeConstraint.MinSize))
+		if volumeConstraint.MinSize == volumeConstraint.MaxSize {
+			return fmt.Errorf("%s total local volume size must be equal to %s", commercialType, min)
 		}
 
-		if localVolumeTotalSize < volumeConstraint.MinSize || localVolumeTotalSize > volumeConstraint.MaxSize {
-			min := humanize.Bytes(uint64(volumeConstraint.MinSize))
-			if volumeConstraint.MinSize == volumeConstraint.MaxSize {
-				return fmt.Errorf("%s total local volume size must be equal to %s", commercialType, min)
-			}
-
-			max := humanize.Bytes(uint64(volumeConstraint.MaxSize))
-			return fmt.Errorf("%s total local volume size must be between %s and %s", commercialType, min, max)
-		}
-	} else {
-		logger.Warningf("unrecognized server type: %s", commercialType)
-		logger.Warningf("skip local volume size validation")
+		max := humanize.Bytes(uint64(volumeConstraint.MaxSize))
+		return fmt.Errorf("%s total local volume size must be between %s and %s", commercialType, min, max)
 	}
 
 	return nil
 }
 
-func validateRootVolume(api *instance.API, zone scw.Zone, image string, rootVolume *instance.VolumeTemplate) error {
+func validateRootVolume(getImageResponse *instance.GetImageResponse, rootVolume *instance.VolumeTemplate) error {
```

Comment:
We don't need the whole `*instance.GetImageResponse` here:
`func validateRootVolume(requiredSize *scw.Size, rootVolume *instance.VolumeTemplate) error`